crypto/evp/pmeth_lib.c

```diff
@@ -355,6 +359,12 @@ int EVP_PKEY_CTX_ctrl(EVP_PKEY_CTX *ctx, int keytype, int optype,
 
 }
 
+int EVP_PKEY_CTX_ctrl_uint64(EVP_PKEY_CTX *ctx, int keytype, int optype,
+                                int cmd, uint64_t value)
```

Comment:
Ah, dang! As requested, new PR: https://github.com/openssl/openssl/pull/4166 -- thanks for the review!